### PR TITLE
Align configs with live version

### PR DESCRIPTION
**Pull request by sergey-korenets-fivestars**

### CODE REVIEW

File: config-example/LOCAL/config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "is_local": true,
+    "is_prod": false,
```
Comment (abandali) [inline config-example/LOCAL/config.json:3]: technically this is not needed as the default is set in `aladdin.sh` if the value extraction fails: https://github.com/fivestars-os/aladdin/blob/1.19.7.14/aladdin.sh#L157-L160

notice that the sample PROD config has the correct setting: https://github.com/fivestars-os/aladdin/blob/1.19.7.14/config-example/CLUSTERPROD/config.json#L6

is there a compelling reason for setting this in this example LOCAL config?

File: config-example/default/config.json [hunk 2]
```diff
@@ -6,5 +6,6 @@
     "cluster_init": [
         {"project": "dashboard", "ref": "v1.0.0", "namespace": "kube-system"}
     ],
-    "dual_dns_prefix_annotation_name": "dns-name"
+    "dual_dns_prefix_annotation_name": "dns-name",
+    "bastion_account_enabled": false
```
Comment (abandali) [inline config-example/default/config.json:10]: good catch !

File: config-example/LOCAL/config.json (hunk 1 above)
Comment (sergey-korenets-fivestars) [reply config-example/LOCAL/config.json]: @abandali , thanks, just verified, it works fine without this option.

**Review by abandali — APPROVED** [review]
LGTM

**Pull request merged**